### PR TITLE
SEOImages: User Story 1463880, Part 17

### PR DESCRIPTION
**Pull request by damabe**
## Summary
Updating image file names and references to improve SEO.

Files #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: 
https://contentqacats.azurewebsites.net/CheckResults/2019031700001/Report/

There is one image name identified by CATS as being too short. It should be handled at a later time as it is art of the SEO image without hyphen work.

File: docs/framework/windows-workflow-foundation/pick-activity.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ The <xref:System.Activities.Statements.Pick> activity simplifies the modeling of
   
  The following screenshot from the [Using the Pick Activity](./samples/using-the-pick-activity.md) SDK sample shows a Pick activity with two branches.  One branch has a trigger called **Read input**, a custom activity that reads input from the command line. The second branch has a <xref:System.Activities.Statements.Delay> activity trigger. If the **Read input** activity receives data before the <xref:System.Activities.Statements.Delay> activity finishes, <xref:System.Activities.Statements.Delay> Delay will be canceled and a greeting will be written to the console.  Otherwise, if the **Read input** activity does not receive data in the allotted time, then it will be canceled and a timeout message will be written to the console.  This is a common pattern used to add a timeout to any action.  
   
- ![Pick Activity](./media/pickconceptual.JPG "PickConceptual")  
+ ![Pick Activity](./media/pick-activity/using-pick-activity-two-branches.jpg)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/pick-activity.md]:
```suggestion
 ![Pick Activity](./media/pick-activity/pick-activity-two-branches.jpg)  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you. Have just one suggestion to not use ing on the image file name.

**Comment by damabe** [comment]
@mairaw Thanks for your help. The change is made. Updated CATS report:

https://contentqacats.azurewebsites.net/CheckResults/2019031900032/Report/

There is one image name identified by CATS as being too short. It should be handled at a later time as it is art of the SEO image without hyphen work.

**Review by mairaw — APPROVED** [review]
Thanks @damabe 🚀

**Pull request merged**